packages/analytics/analytics-chart/src/types/chart-data.ts

```diff
@@ -60,7 +60,15 @@ export interface AnalyticsChartOptions {
   /**
    * Chart dataset color palette
    */
-  chartDatasetColors?: AnalyticsChartColors | string[]
+  chartDatasetColors?: AnalyticsChartColors | string[],
+  /**
+   *
+   */
+  metricDisplay?: ChartMetricDisplay,
+  /**
+   * Determines if a simplified version of the chart should be displayed
+   */
+  isSimple?: boolean,
```

Comment:
These options only pertain to the new SimpleChart class of charts

`metricAxesTitle` `dimensionAxesTitle` `stacked` do not pertain to the simple chart type as far as I can tell so far.

I would make a separate interface called `SimpleChartOptions` that contains only the options pertaining to `SimpleCharts`